src/main/java/dk/statsbiblioteket/doms/licensemodule/service/LicenseModuleResource.java

```diff
@@ -56,8 +56,8 @@ public CheckAccessForIdsOutputDTO checkAccessForIdsJSON(CheckAccessForIdsInputDT
 	
 	
 	@POST	
-	@Consumes(MediaType.TEXT_XML)	
-	@Path("checkAccessForIds")	
+	@Path("checkAccessForIds")
+	@Consumes(MediaType.TEXT_XML)			
 	@Produces(MediaType.TEXT_XML)
```

Comment:
As Asger noted, consider adding MediaType.APPLICATION_XML to the list of acceptable Consumes and Produces

Reply:
Done